releases/release-4.0.9.md

```diff
@@ -0,0 +1,168 @@
+---
+title: TiDB 4.0.9 Release Notes
+---
+
+# TiDB 4.0.9 Release Notes
+
+Release date: December 3, 2020
+
+TiDB version: 4.0.9
+
+## New Features
+
++ TiFlash
+
+    - Support storing the latest data of the storage engine on multiple disks [#1159](https://github.com/pingcap/tics/pull/1159)
+
+## Improvements
+
++ TiDB
+
+    - Ban (index) merge join heuristically when converting equal conditions to other conditions. [#21146](https://github.com/pingcap/tidb/pull/21146)
+    - Differentiate types for user variables [#21107](https://github.com/pingcap/tidb/pull/21107)
+    - Allow set GOGC in the config file. [#20922](https://github.com/pingcap/tidb/pull/20922)
+    - Make dump binary time more compatible with MySQL [#21135](https://github.com/pingcap/tidb/pull/21135)
+    - TiDB now provides an error for statements that use the syntax `LOCK IN SHARE MODE`. This helps alert users that this feature is not implemented by TiDB, but a workaround to restore the previous behavior is available by setting `tidb_enable_noop_functions=1`. [#21005](https://github.com/pingcap/tidb/pull/21005)
+    - Avoid unnecessary warnings/errors when folding constants in shortcut-able expressions [#21040](https://github.com/pingcap/tidb/pull/21040)
+    - Raise an error when preparing the `load data` statement. [#21199](https://github.com/pingcap/tidb/pull/21199)
+    - Ignore integer zero-fill size attribute when changing the column types [#20986](https://github.com/pingcap/tidb/pull/20986)
+    - Add runtime information for DML in explain analyze [#21066](https://github.com/pingcap/tidb/pull/21066)
+    - Disallow multi-updates on the primary key (i.e. updating primary key of the same table with different alias) [#21113](https://github.com/pingcap/tidb/pull/21113)
+    - Add metric for connection idle time [#21301](https://github.com/pingcap/tidb/pull/21301)
+
++ TiKV
+
+    - Add tag to split command to trace its source [#8936](https://github.com/tikv/tikv/pull/8936)
+
```

Comment:
```suggestion
```suggestion
+ PD
    - Check cluster version on stores become tombstone [#3213](https://github.com/pingcap/pd/pull/3213)
    - The store of lower version is no longer allowed to change from `Tombstone` back to `Up`. [#3206](https://github.com/pingcap/pd/pull/3206)
```
```